test/integration/dockerfile/dockerfile_test.go

```diff
@@ -1,590 +1,22 @@
 // +build integration
 
-package integration
+package dockerfile
```

Comment:
This wasn't a rename - I moved all of the Dockerfile test code to a separate package.